osclib/core.py

```diff
@@ -312,16 +326,10 @@ def devel_project_get(apiurl: str, target_project: str, target_package: str) ->
             raise e
 
     if target_project.endswith('openSUSE:Factory'):
-        try:
-            headers = {'Accept': 'text/plain'}
-            with http_GET('https://src.opensuse.org/openSUSE/Factory/raw/branch/main/pkgs/_meta/devel_packages', headers=headers) as f:
-                for line in f:
-                    pkg, _, prj = line.decode('utf-8').strip().partition(' ')
-                    if pkg == target_package:
-                        return prj, pkg
-        except HTTPError as e:
-            if e.code != 404:
-                raise e
+        devel_pkgs = git_factory_devel_project_dict_get(apiurl)
+        devel_prj = devel_pkgs.get(target_package)
+        if devel_prj is not None:
+            return devel_prj, target_package
```

Comment:
you could also do something like:

```suggestion
            if target_package in devel_pkgs:
                 return devel_pkgs[target_package], target_package
                 
```